src/Products/Annotation/Controllers/AnnotationApiController.cs

```diff
@@ -30,365 +27,273 @@ namespace GroupDocs.Annotation.WebForms.Products.Annotation.Controllers
     [EnableCors(origins: "*", headers: "*", methods: "*")]
     public class AnnotationApiController : ApiController
     {
-        private static Common.Config.GlobalConfiguration globalConfiguration = new Common.Config.GlobalConfiguration();
-        private static AnnotationImageHandler annotationImageHandler;
-        private readonly List<string> supportedImageFormats = new List<string> { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png", ".gif", ".emf", ".wmf", ".dwg", ".dicom", ".djvu" };
-        private readonly List<string> supportedDiagrammFormats = new List<string> { ".vsd", ".vdx", ".vss", ".vsx", ".vst", ".vtx", ".vsdx", ".vdw", ".vstx", ".vssx" };
+        private static Common.Config.GlobalConfiguration GlobalConfiguration;
+        private readonly List<string> SupportedImageFormats = new List<string> { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png", ".dwg", ".dcm", ".dxf" };
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="AnnotationApiController"/> class.
-        /// Constructor.
+        /// Constructor
         /// </summary>
         public AnnotationApiController()
         {
-            // create annotation directories
-            DirectoryUtils directoryUtils = new DirectoryUtils(globalConfiguration.Annotation);
-
-            // create annotation application configuration
-            AnnotationConfig config = new AnnotationConfig
-            {
-                // set storage path
-                StoragePath = directoryUtils.FilesDirectory.GetPath(),
-            };
-
-            // initialize Annotation instance for the Image mode
-            annotationImageHandler = new AnnotationImageHandler(config);
+            GlobalConfiguration = new Common.Config.GlobalConfiguration();
```

Comment:
Codacy found an issue: [Remove this assignment of 'GlobalConfiguration' or initialize it statically.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-WebForms/pullRequest?prid=5483022)